akka-actor/src/main/scala/akka/actor/LightArrayRevolverScheduler.scala

```diff
@@ -252,7 +258,8 @@ class LightArrayRevolverScheduler(config: Config,
 
     @tailrec final def nextTick(): Unit = {
       val time = clock()
-      val sleepTime = start + (tick * tickNanos) - time
+      val t = if (tick >= 0) tick else Int.MinValue + tick
+      val sleepTime = start + ((tickWrap + t) * tickNanos) - time
```

Comment:
hm, this is not completely correct, either. The intention was that `tickWrap + t` should represent the total number of ticks.

I think we should either change `tick` to a long, or have a separate long tick counter for this instead of this mess.

The scary part with `tick: Long` is here: https://github.com/akka/akka/blob/wip-20424-tick-wrap-patriknw/akka-actor/src/main/scala/akka/actor/LightArrayRevolverScheduler.scala#L220